jest.config.js

```diff
@@ -1,70 +1,32 @@
 module.exports = {
-
   collectCoverage: true,
-
-  collectCoverageFrom: [
-
-    '**/*.js',
-
-    '!**/*.test.js'
-
-  ],
-
+  collectCoverageFrom: ['**/*.mjs', '!**/*.test.mjs'],
   coverageDirectory: 'test-output',
-
-  coverageReporters: [
-
-    'text-summary',
-
-    'lcov'
-
-  ],
-
+  coverageReporters: ['text-summary', 'lcov'],
   coveragePathIgnorePatterns: [
-
     '<rootDir>/node_modules/',
-
     '<rootDir>/test-output/',
-
     '<rootDir>/test/',
-
     '<rootDir>/jest.config.js'
-
-  ],
-
-  modulePathIgnorePatterns: [
-
-    'node_modules'
-
   ],
-
+  modulePathIgnorePatterns: ['node_modules'],
   reporters: [
-
     'default',
-
     [
-
       'jest-junit',
-
       {
-
         suiteName: 'jest tests',
-
         outputDirectory: 'test-output',
-
         outputName: 'junit.xml'
-
       }
-
     ]
-
   ],
-
   testEnvironment: 'node',
-
   testPathIgnorePatterns: [],
-
   setupFilesAfterEnv: ['./jest.setup.js'],
-
-  verbose: true
+  verbose: false,
```

Comment:
Do we not want to keep the verbose output?

Reply:
Ah yeah, I'll put this back. I think I was trying to fix an issue, and this might not have been the cause.